game/server/debuggeometries/routes.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+from uuid import UUID
+
+from fastapi import APIRouter, Depends
+
+from game import Game
+from game.server import GameContext
+from .models import HoldZonesJs, IpZonesJs, JoinZonesJs
+from ...ato import Flight
+
+JsonDict = dict[str, Any]
+
+
+router = APIRouter(prefix="/debug/waypoint-geometries")
+
+
+def find_flight(game: Game, flight_id: UUID) -> Flight:
+    for coalition in game.coalitions:
+        for package in coalition.ato.packages:
+            for flight in package.flights:
+                if flight.id == flight_id:
+                    return flight
+    raise KeyError(f"No flight found with ID {flight_id}")
+
+
+@router.get("/hold/{flight_id}")
```

Comment:
Do you think these might expand to `router.get("/something/{package_id}"` at some point?

Reply:
Yeah, eventually there will be endpoints for all the things. The flight is just the thing I needed for this.